plugins/ftdidmx/FtdiDmxDevice.cpp

```diff
@@ -64,7 +65,15 @@ bool FtdiDmxDevice::StartHook() {
     FtdiInterface *port = new FtdiInterface(m_widget,
                                             static_cast<ftdi_interface>(i));
     if (port->SetupOutput()) {
-      AddPort(new FtdiDmxOutputPort(this, port, i, m_frequency));
+      AddPort(new FtdiDmxOutputPort(
+                    this,
+                    port,
+                    i,
+                    m_frequency,
+                    static_cast<unsigned int>(strtol(
```

Comment:
an 8 character base36 (the FTDI serial) string exceeds 32bits of data (it's 40-something bits) so it can have some collisions (though I expect that to be highly unlikely)

Reply:
Changed, will push in a bit with a whole slew of other changes.